src/database/queries.ts

```diff
@@ -103,28 +104,28 @@ export const getAllVoucherRanges = async (): Promise<VoucherRange[]> => {
 /**
  * Query the `voucher-ranges` collection.
  *
- * Return a VoucherRange if the given serialNumber is in any voucher range.
+ * Returns a voucherRange if the given serialNumber is in any voucher range.
```

Comment:
Nit: capitalizing VoucherRange (how you originally had it) would be good since it's the official type name!

Reply:
Should be good to merge after making this change!